Components/WebService/Rest/RestConfigurableProtocol.php

```diff
@@ -15,6 +15,8 @@ class RestConfigurableProtocol extends ConfigurableWebserviceProtocol implements
     const OPTION_HEADERS = 'headers';
     const OPTION_AUTH = 'authentication';
     const OPTION_ENCODING = 'encoding';
+    /** @deprecated: Will be removed on next major version. Fallback will not happen in v3. */
+    const OPTION_REQUEST_FALLBACK_ON_ERROR = 'request_fallback';
```

Comment:
In this case I think the value of the constant to be more accurate should be `response_fallback` because we are not doing any fallback with the request
Also based on that I would rename the constant to `OPTION_RESPONSE_FALLBACK_ON_ERROR'`

Reply:
Agreed. Updated on both ends.